--- gtk+-1.2.10/gtk-config.in.multilib	2007-01-11 10:22:17.000000000 -0500
+++ gtk+-1.2.10/gtk-config.in	2007-01-11 10:33:01.000000000 -0500
@@ -5,8 +5,11 @@
 glib_thread_libs="@glib_thread_libs@"
 glib_thread_cflags="@glib_thread_cflags@"
 
-prefix=@prefix@
-exec_prefix=@exec_prefix@
+[ -z "$PKG_CONFIG" ] && PKG_CONFIG="pkg-config"
+prefix=`${PKG_CONFIG} --variable prefix gtk+`
+exec_prefix=`${PKG_CONFIG} --variable exec_prefix gtk+`
+libdir=`${PKG_CONFIG} --variable libdir gtk+`
+includedir=`${PKG_CONFIG} --variable includedir gtk+`
 exec_prefix_set=no
 
 usage()
@@ -91,14 +94,14 @@
 fi
 
 if test "$echo_cflags" = "yes"; then
-      echo -I@includedir@/gtk-1.2 $glib_cflags @x_cflags@ 
+      echo -I${includedir}/gtk-1.2 $glib_cflags @x_cflags@ 
 fi
 
 if test "$echo_libs" = "yes"; then
       my_glib_libs=
-      libdirs=-L@libdir@
+      libdirs=-L${libdir}
       for i in $glib_libs ; do
-        if test $i != -L@libdir@ ; then
+        if test $i != -L${libdir} ; then
           if test -z "$my_glib_libs" ; then
             my_glib_libs="$i"
           else
